metrics/sources/summary/summary.go

```diff
@@ -43,13 +43,23 @@ var (
 		},
 		[]string{"node"},
 	)
+	scrapeTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "metrics_server",
+			Subsystem: "kubelet_summary",
+			Name:      "scrapes_total",
+			Help:      "Total number of attempted Summary API scrapes done by Metrics Server",
+		},
+		[]string{"success"},
```

Comment:
As an optional, success label forces us to use just two states. Maybe replacing it with response code from the request is going to be better approach.

Reply:
We can always add a new label if it turns out we need it. For now I'm merging as it is.